sourcegraph/sourcegraph
# search: use selected context for homepage examples

https://user-images.githubusercontent.com/6417322/195516865-ce8b756f-3e1d-401a-8f9a-27c5b127393d.mov

## Test plan



* Open search homepage
* Switch search context
* Homepage examples should reflect the currently selected context

## App preview:

- [Web](https://sg-web-rn-query-examples-homepage-context.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-slutvcqvdc.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
